README.md

```diff
@@ -113,12 +113,7 @@ on your Trino instance.
 
 #### Supported Functionality
 
-Due to the nature of Trino, not all core `dbt` functionality is supported.
-The following features of dbt are not implemented in `dbt-trino`:
-
-- Snapshot
-
-Also, note that upper or mixed case schema names will cause catalog queries to fail.
+Note that upper or mixed case schema names will cause catalog queries to fail.
```

Comment:
I've tested and it seems to be supported. TBH, I'm not sure whether it was an issue before, since I didn't face it.

Reply:
We recently experienced this actually. I think it is because metadata is converted to lowercase causing the scheme not to be found. I will see if I can reproduce.